wgengine/router/ifconfig_windows.go

```diff
@@ -386,13 +387,18 @@ func configureInterface(cfg *Config, tun *tun.NativeTun) error {
 
 // isMissingIPv6Err reports whether err is due to IPv6 not being enabled on the machine.
 //
-// It's intended for use on errors returned by the winipcfg.Interface.GetIpInterface
-// method, which ultimately calls:
-// https://docs.microsoft.com/en-us/windows/win32/api/netioapi/nf-netioapi-getipinterfaceentry
+// It only currently supports the errors returned by winipcfg.Interface.GetIpInterface.
 func isMissingIPv6Err(err error) bool {
-	// ERROR_NOT_FOUND from means the address family (IPv6) is not found.
-	// (ERROR_FILE_NOT_FOUND means that the interface doesn't exist.)
-	return errors.Is(err, windows.ERROR_NOT_FOUND)
+	if se, ok := err.(*os.SyscallError); ok {
+		switch se.Syscall {
+		case "iphlpapi.GetIpInterfaceEntry":
```

Comment:
This gets complicated fast. Can we adjust the function to return the error we want instead?

Reply:
https://github.com/tailscale/winipcfg-go is already a fork we want to reconcile with upstream. Adding more differences would make that job harder. I filed https://github.com/tailscale/tailscale/issues/760 ... want to work on that?

In the meantime I want to minimize touching it.

Reply:
> I filed #760 ... want to work on that?

Sure. I will work on that next.

LGTM in the meantime.

Alex